.github/labeler.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2020 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pr-docker:
+- 'tools/dockerbuild/**'
+- 'gradle/wrapper/**'
+- '.github/**'
+
+pr-helm:
+- 'helm/**'
+- '.github/**'
+
+pr-native:
+- 'gradle/wrapper/**'
+- 'servers/**'
+- '.github/**'
+
+pr-integrations:
```

Comment:
We should probably add `pr-integrations` also in case of `model` changes, WDYT?

Reply:
Sure, mind opening a PR?

Reply:
#6132